ONSdigital/dockers
# Modify docker autobuilds to use ubuntu over alpine

**Changes**
- Use ubuntu's image as the base os, rather than alpine linux
- Increases size of docker images by about 100MB
- Allows use of docker-compose locally as file sync changes between virtualbox are not an issue

**How to test**
- Follow README.md instructions for installing image and test against sde-console
- Make a change to server.py in sde-console (eg. use app.logger to log something) and confirm it auto reloads and reflects changes

**Who can test**
- Anyone but @iwootten
